migration/20181220-register-isbns-for-oclc-coverage.sql

```diff
@@ -4,13 +4,15 @@ delete from coveragerecords where id in (
  select cr.id from coveragerecords cr join datasources ds on cr.data_source_id=ds.id join identifiers i on cr.identifier_id=i.id where ds.name='OCLC Classify' and operation is null and i.type='ISBN'
 );
 
--- For every ISBN associated with a collection, create a coverage
--- record in the 'registered' state for the OCLC CLassify coverage
--- provider.
+-- For every ISBN associated with a collection and not already
+-- registered, create a coverage record in the 'registered' state for
+-- the OCLC CLassify coverage provider.
 insert into coveragerecords (
  identifier_id, data_source_id, timestamp, status
 ) select 
- i.id, ds.id, now(), 'registered'
+ distinct(i.id), ds.id, now(), coverage_status('registered')
```

Comment:
The distinct() is what was missing from the original script. The rest is something I added to make it safer to add coverage records without creating duplicates.